doc/whats_new/v0.21.rst

```diff
@@ -52,6 +52,17 @@ Support for Python 3.4 and below has been officially dropped.
   graph, which would add explicitly zeros on the diagonal even when already
   present. :issue:`12105` by `Tom Dupre la Tour`_.
 
+:mod:`sklearn.neighbors`
+......................
+
+- |Fix| Fixed a bug in :class:`sklearn.neighbors` calling functions
+  :func: `NearestNeighbors.kneighbors`,
+  :func: `NearestNeighbors.radius_neighbors`,
+  :func: `NearestNeighbors.kneighbors_graph`,
+  :func: `NearestNeighbors.radius_neighbors_graph`
```

Comment:
I would suggest building the docs locally and checking the formatting is correct. Here you need to remove spaces after :func: